group_vars/all.sample

```diff
@@ -260,6 +260,8 @@ dummy:
 ## MDS options
 #
 #mds_use_fqdn: false # if set to true, the MDS name used will be the fqdn in the ceph.conf
+#mds_allow_multimds: false
```

Comment:
One last thing, please update roles/ceph-common/defaults/main.yml instead and run `generate_group_vars_sample.sh`, thanks